theme/brb/views/authentication/home.hbs

```diff
@@ -5,12 +5,14 @@ mode="append"}}
 
 {{/content}} {{#content "body"}} {{#embed "lib/header" hideMenu=true}}{{/embed}}
 <div class="container">
-	<div class="row">
+	<div class="row" style="flex-direction: row-reverse;">
```

Comment:
We do not want inline css anymore. Please rethink if this is absolutely necessary here.